tests/unit/ui/components/TestHeroSection.js

```diff
@@ -29,14 +28,12 @@ describe(__filename, () => {
   });
 
   it('renders a Link if linkTo prop is supplied', () => {
-    const root = shallowRender({ linkTo: '/whatever/' });
+    const linkTo = '/somewhere/';
+    const root = shallowRender({ linkTo });
     const link = root.find(Link);
 
     expect(link).toHaveProp('className', 'HeroSection-link-wrapper');
-    expect(link).toHaveProp(
-      'to',
-      `/whatever/?src=${INSTALL_SOURCE_HERO_PROMO}`,
-    );
```

Comment:
Since this has been removed, it is important to add a test case in the test file of `HomeHeroBanner`.